CODE_OF_CONDUCT.md

```diff
@@ -1 +0,0 @@
-./docs/CODE_OF_CONDUCT.md
```

Comment:
Hi, @major-winter. These changes are not relevant to the actual PR. Please remove these changes. The same goes for other `.md` files.